sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -35,7 +35,7 @@ string name = widget.Name;
 
 ### Set a JSON property
 
-JSON members can be set on the dynamic object.
+JSON members can be set on the dynamic object.  Pass `DynamicCaseMapping.PascalToCamel` to `ToDynamicFromJson()` to write JSON members with camelCase names.
```

Comment:
Should we mention why this would be desirable?   Specifically, I'm thinking it may help to mention that idiomatic JSON uses camelCase and Azure services are expected to treat data in a case-sensitive manner.

Reply:
Cool, will add this.